PowerUp/app/src/main/res/layout/about.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:background="@drawable/background"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:weightSum="1">
+    <TextView
+        android:layout_width="200dp"
+        android:layout_height="250dp"
+        android:text="@string/about_text"
+        android:id="@+id/textView"
+        android:fontFamily="cursive"
+        android:typeface="serif"
+        android:layout_alignParentTop="true"
+        android:layout_weight="0.29"
+        android:textSize="16dp"
+        android:textIsSelectable="false"
+        android:textColor="#000000" />
+
+    <ImageButton
+        android:id="@+id/startButtonMain1"
```

Comment:
I would suggest a better name for the button id that actually explains what it is and what it does